Azure/azure-cli
# [ARO] `az aro`: v20230401 API updates

**Related command**
`az aro`

**Description**
- Update `azure-mgmt-redhatopenshift` SDK to 1.3.0, providing ARO v20230401 API
- Add `--outbound-type` param to `az aro create` command
- Add `az aro validate` command to pre-validate cluster creation parameters
- Provide the above validation during `az aro create`

**Testing Guide**
- Verify all ARO API calls are to v20230401 API
- Ensure `--outbound-type` flag is present and usable on `az aro create`
- Ensure standalone `az aro validate` command, as well as validation on `az aro create`, function as expected

**History Notes**


[ARO] `az aro` (all commands): Update to v20230401 API
[ARO] `az aro create`: Add new `--outbound-type` parameter, allowing users to select "Loadbalancer" (default) or "UserDefinedRouting"
[ARO] `az aro create`: Perform pre-flight validation of prerequisite permissions before creation
[ARO] `az aro validate`: New command to perform explicit validation of prerequisite permissions

---

This checklist is used to make sure that common guidelines for a pull request are followed.
